backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/DTOs/AddressDTO.java

```diff
@@ -0,0 +1,12 @@
+package com.nokia.library.nokiainnovativeproject.DTOs;
+
+import lombok.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AddressDTO {
```

Comment:
There are no constraints on the fields in any DTO. Now there is possibility to save nulls to database. 
Also in every Controller this constraints should be checked.